internal/config/config_file_test.go

```diff
@@ -150,20 +153,149 @@ func Test_parseConfigFile(t *testing.T) {
 
 func Test_ConfigDir(t *testing.T) {
 	tests := []struct {
-		envVar string
-		want   string
+		name        string
+		onlyWindows bool
+		env         map[string]string
+		output      string
 	}{
-		{"/tmp/gh", ".tmp.gh"},
-		{"", ".config.gh"},
+		{
+			name:   "no envVars",
+			output: ".config.gh",
+			env: map[string]string{
+				"AppData": "",
+			},
+		},
+		{
+			name: "GH_CONFIG_DIR specified",
+			env: map[string]string{
+				"GH_CONFIG_DIR": "/tmp/gh_config_dir",
```

Comment:
On Windows, it's unlikely that a path set by environment variable would have slashes. It's more likely that it would have backslashes. You can normalize this via
```suggestion
				"GH_CONFIG_DIR": filepath.FromSlash("/tmp/gh_config_dir"),
```